evap/evaluation/models.py

```diff
@@ -410,11 +418,11 @@ def responsible_contributors(self):
 
     @property
     def days_left_for_evaluation(self):
-        return (self.vote_end_date - datetime.date.today()).days
+        return (self.vote_end_date - date.today()).days
```

Comment:
as per @janno42's request, can you change this to only take days into account, not the time?